src/dss/LerpFactoryAbstract.sol

```diff
@@ -7,7 +7,7 @@ interface LerpFactoryAbstract {
     function rely(address) external;
     function deny(address) external;
     function lerps(bytes32) external view returns (address);
-    function active(uint256) external view returns (address);
+    function active(uint256) external view returns (address[] memory);
```

Comment:
Default view for an array is indexed getter instead of returning the whole array, so the original is correct in this case.

Reply:
Yep, you're right.
Done in 1401821.